src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -113,7 +113,7 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     DepositDir.get(draftBase, user, id)
   }
 
-  def getUser(user: String): Try[Map[String, Seq[String]]] = authentication.getUser(user)
+  def getUserProperties(user: String): Try[Map[String, Seq[String]]] = authentication.getUser(user)
```

Comment:
I'm not quite happy with the return type here. I know it didn't really change in this PR, but why not do the parsing of the `Map` here or in `authentication.getUser` (latter is even better, IMO)? By returning a `User` object, you don't have to parse it all the way down in `AgreementsXml` and hence that class doesn't need to know about the LDAP specific keywords.